### DISCUSSION TITLE
Question about upgrading the version for Unity.

### DISCUSSION THREAD

**Discussion by JHsGame** [fo-dicom/fo-dicom#1346] [Q&A]
Nice to meet you all. Good day.

Please understand that I use a translator because I don't speak English.

What I'm thinking about now is that the latest version of fo-Dicom is 5.0, while the version of fo-Dicom asset for Unity is 3.5.

So I want to version up the fo-Dicom used in Unity. Does anyone know what to do?

**Comment by JHsGame** [comment]
@gofal @amoerie
I heard that you two worked on Unity. Can I know how?

**Comment by amoerie** [comment]
I don't have experience with Unity at all, sorry.

**Reply by JHsGame** [reply]
Oh... Someone tagged me because they thought you had experienced it, but it wasn't.

Does anyone have experience with Unity or know how to upgrade the Unity version? I beg of course.

**Comment by gofal** [comment]
Me too, i have no experience with Unity. The Unity package used to be a complete separate package built by Anders Gustafsson. When he stopped working on fo-dicom he published his sources to anyone interested in it. So the fo-dicom package for unity is not mainatained officially any more.

**Reply by JHsGame** [reply]
Then, is there anyone who knows? I would be very grateful if you could give me any information about the published code. I really want to solve the version problem.

**Reply by gofal** [reply]
So you already downloaded the code from here: https://github.com/fo-dicom/fo-dicom/discussions/1248 ?
Then I would search in issues for all open and closed issues that contain the word "unity" - there you might find some hints.

**Reply by JHsGame** [reply]
Thank you for your answer. It won't be easy because I'm a beginner in this field, but I'll refer to it.

**Comment by JHsGame** [comment]
@anders9ustafsson
Is Mr.Anders Gustafsson here? I heard that you worked with Unity. I'm writing this because I was wondering if you could let me know about upgrading the version.
